src/plugins/hiddenColumns/hiddenColumns.js

```diff
@@ -233,6 +234,21 @@ class HiddenColumns extends BasePlugin {
     this.hideColumns(column);
   }
 
+  /**
+   * Returns an array of physical indexes of hidden columns.
+   *
+   * @returns {Number[]}
+   */
+  getHiddenColumns() {
+    return this.hiddenColumnsMap.getValues().reduce((hiddenColumns, flag, index) => {
```

Comment:
Why not use `Array.filter` instead `Array.reduce`?

Reply:
Values are booleans. We need indexes (native array indexes represent physical indexes).